.github/workflow/ISSUE_TEMPLATE/documentation.yml

```
@@ -0,0 +1,34 @@
+name: 📄 Documentation issue
+description: Found an issue in the documentation? You can use this one!
+title: "[DOCS] <description>"
```

```suggestion
title: "[DOCS]:"
```
You can remove ` <description>`  from the title and add a colon after brackets.